.github/workflows/release.yaml

```diff
@@ -1,21 +1,40 @@
 ---
-name: Release Container Image
+name: Release
 
 on:
   push:
     branches: [main]
-    tags:
-      - "v*.*.*"
+  workflow_call:
+    secrets:
+      ADFINISBOT_PAT:
+        required: true
 
 jobs:
-  container:
+  semrel:
+    name: Semantic Release
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
+      - name: Semantic Release
+        uses: go-semantic-release/action@v1
+        id: semrel
+        with:
+          github-token: ${{ secrets.ADFINISBOT_PAT }}
+
+      - name: Setup Poetry
+        if: steps.semrel.outputs.version != ''
+        uses: abatilo/actions-poetry@v3.0.0
```

Comment:
did you consider just running `pipx install poetry` instead of this action?

Reply:
i adjusted it to use pipx